src/azure-cli/azure/cli/command_modules/resource/commands.py

```diff
@@ -291,6 +297,13 @@ def load_command_table(self, _):
         g.custom_command('show-log', 'get_deployment_script_logs')
         g.custom_command('delete', 'delete_deployment_script', confirmation=True)
 
+    with self.command_group('template-specs', resource_templatespecs_sdk, resource_type=ResourceType.MGMT_RESOURCE_TEMPLATESPECS, is_preview=True) as g:
+        g.custom_command('create', 'create_or_update_template_spec', validator=_validate_template_input)
+        g.custom_command('export', 'export_template_spec', validator=_validate_template_spec_out)
+        g.custom_show_command('get', 'get_template_spec', validator=_validate_template_spec)
```

Comment:
Usually, CLI use 'show' as command name instead of 'get'.

Reply:
Only renaming command as show is enough, method name can still keep "get_template_spec".

---
In reply to: [465443074](https://github.com/Azure/azure-cli/pull/14448#discussion_r465443074) [](ancestors = 465443074)